site/content/contribute/more-info/server/rest-api.md

```diff
@@ -19,33 +19,36 @@ Looking for the API reference? You can find it here: {{<newtabref title="https:/
 To add an endpoint to API version 4, all of the following must be completed:
 
 - [Document the endpoint](#document-the-endpoint)
-- [Implement the API handler on the server](#implement-the-api-handler)
+- [Implement the API handler](#implement-the-api-handler)
 - [Add a function to the Go driver](#update-the-golang-driver)
 - [Write a unit test](#write-a-unit-test)
 - [Submit your implementation!](#submit-your-pull-request-pr)
 
-A full example of the process can be found through these two pull requests:
+#### Document the endpoint
+At Mattermost, the [OpenAPI specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md) is used for API documentation. The API documentation lives in the main Mattermost repository alongside the server: [api](https://github.com/mattermost/mattermost/tree/master/api).
 
-- Documenting the `POST /teams` endpoint: {{< newtabref href="https://github.com/mattermost/mattermost-api-reference/pull/72" title="mattermost-api-reference#72" >}}
-- Implementing the `POST /teams` endpoint: {{< newtabref href="https://github.com/mattermost/mattermost/pull/5220" title="mattermost/server#5220" >}}
+To document an endpoint, follow these steps:
 
-#### Document the endpoint
-At Mattermost, the [OpenAPI specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md) is used for API documentation. The API documentation lives in the [mattermost-api-reference](https://github.com/mattermost/mattermost-api-reference) repository. To document an endpoint, follow these steps:
+1. Create a branch for your changes.
```

Comment:
nit: This can be interpreted in a way that they need to create a new branch for **any** API change. But if we consider that CI checks works properly, they won't be able to merge a PR that changes an endpoint. Which means they need to update the API spec in their actual branch.

Reply:
I'm open to omitting this step altogether if that would clarify things. I think it only existed previously because of t need to submit two PRs, but it's somewhat redundant to explain how to submit a PR with this level of detail.

Reply:
Thanks, I agree it's a bit confusing to have it here.